pyrasa/utils/aperiodic_utils.py

```diff
@@ -215,16 +225,25 @@ def num_zeros(decimal):
     return df_aps, df_gof
 
 
-def compute_slope_sprint(aperiodic_spectrum, freqs, times, fit_func, ch_names=[], fit_bounds=None):
+def compute_slope_sprint(
+    aperiodic_spectrum: np.ndarray,
+    freqs: np.ndarray,
+    times: np.ndarray,
+    fit_func: str,
+    ch_names: Iterable = [],
```

Comment:
Never use mutable types (lists, dicts, etc) as default arguments. Replace this by a tuple.

Reply:
should I also remove the iterable annotation?

Reply:
no, the annotation is fine. because when I call the function, it's totally fine to provide any kind of iterable, including a `list`.

But assigning one as a default argument means that if you did something like this in your code:
```python
ch_name.add('superchannel')
```

the next call to this function would have the modified list (including the `superchannel` entry) as a default.

this cannot happen with non-mutable types.

If you need this to be an empty list by default, the safe pattern is:

```python
def my_function(this_list: Optional[Iterable]=None):
    if this_list is None:
        this_list = []
```

Reply:
ok makes sense